tests/unit/tuner/callback/test_early_stopping.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import paddle
+import pytest
+import tensorflow as tf
+import torch
+
+from finetuner.tuner.base import BaseTuner
+from finetuner.tuner.callback import EarlyStopping
+from finetuner.tuner.keras import KerasTuner
+from finetuner.tuner.paddle import PaddleTuner
+from finetuner.tuner.pytorch import PytorchTuner
+from finetuner.tuner.state import TunerState
+
+
+@pytest.fixture(scope='module')
+def pytorch_model() -> BaseTuner:
+    embed_model = torch.nn.Sequential(
+        torch.nn.Linear(
+            in_features=10,
+            out_features=10,
+        ),
+    )
+    return embed_model
+
+
+@pytest.fixture(scope='module')
+def keras_model() -> BaseTuner:
+    embed_model = tf.keras.Sequential(
+        [
+            tf.keras.layers.Dense(10),
```

Comment:
Same here, will be more compact (but here you can't remove sequential)
```suggestion
            tf.keras.layers.Dense(10)
```